src/server/test/web/readingsCompareMeterQuantity.js

```diff
@@ -158,8 +158,114 @@ mocha.describe('readings API', () => {
 
 				// Add C10 here
 
-				// Add C11 here
-
+				mocha.it('C11: 1 day shift end 2022-10-31 17:00:00 for 15 minute reading intervals and quantity units & kWh as BTU reverse conversion', async () => {
+					const unitData = unitDatakWh.concat([
+						// adding units u1, u2, u3, u16
+						{
```

Comment:
I'm sorry if this was not clear. u1 & u2 are not needed as unitDatakWh already include them. These two can be deleted with u3 & u16 left.